tools/d2g/d2gtest/d2g_test.go

```diff
@@ -33,7 +33,11 @@ func ExampleScopes_mixture() {
 		"docker-worker:capability:device:loopbackVideo:x/y/z",
 		"docker-worker:capability:device:kvm:x/y/z",
 	}
-	gwScopes := d2g.Scopes(dwScopes)
+	gwScopes, err := d2g.Scopes(dwScopes, nil)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
```

Comment:
I think this doesn't work for full task definition conversions?

Reply:
This is just a simple test to see that the scopes convert. Isn't that ok?